geopandas/explore.py

```diff
@@ -285,6 +299,13 @@ def my_colormap(value):  # scalar value defined in 'column'
     except (ImportError, ModuleNotFoundError):
         HAS_XYZSERVICES = False
 
+    # isolate MPL version - GH#2596
+    mpl = Version(matplotlib.__version__)
+    if mpl >= Version("3.6"):
+        MPL_36 = True
+    else:
+        MPL_36 = False
```

Comment:
```suggestion
    MPL_36 = Version(matplotlib.__version__) >= Version("3.6")
```

this should be enough and saves space and complexity.